src/classes/RD2_OpportunityEvaluationService_TEST.cls

```diff
@@ -1834,9 +1834,32 @@ private class RD2_OpportunityEvaluationService_TEST {
         System.assertEquals(0, errors.size(), 'No error should be created: ' + errors);
     }
 
+    /**
+    * @description Verifies that only fields in Custom Field Mapper or is necessary for calculation should be included
+    */
+    @IsTest
+    private static void shouldOnlyQueryNecessaryAndCustomFieldMappingFields() {
+        RD2_CustomFieldMapper_TEST.createCustomFieldMapping(
+            String.valueOf(npe03__Recurring_Donation__c.ClosedReason__c)
```

Comment:
"Status Reason" (`ClosedReason__c`) is already in the list of mandatory fields. Is there any other field we can choose to be part of the custom field mapping so we can verify the mapped field is included in the query?